### ISSUE TITLE
Add FileSaver to dependencies or mention it in docs

### ISSUE THREAD

**Issue by arogachev** [evidenceprime/html-docx-js#23]
Example in docs contains usage of `saveAs ` function which belongs to [FileSaver.js](https://github.com/eligrey/FileSaver.js) extension. Without looking at the demo page sources, it's impossible to figure out why example is not working.

So I suggest to either add it to dependencies or at least mention in the docs.

First option is more preferable for me, because file saving it's important part of converting html to docx. Can we do something with converted object without file saving?

**Comment by avk** [comment]
Still an issue in December.

**Comment by N1kto** [comment]
Sorry guys for no feedback for a long time.
I think `filesaver.js` dependency is not mentioned anywhere because *saving files* is not in scope of this package. `html-docx` package is about converting HTML into docx zip `blob` or `buffer`. How you eventually consume the generated `blob/buffer` depends on your particular use case. This package is polymorphic meaning you can use it in `node.js` environment and there you don't need any file saver - you simply write `buffer` into file with native node methods. And even coming back to `client` environment, more and more browsers implement `download` attribute support for anchor elements, so eventually `filesaver.js` will not be needed OR any other alternative solution may come out - should we list all of those? I doubt it is a right thing to do. We used that package  for demo only.

@anowak what do you think?

**Comment by anowak** [comment]
Yes, this was indeed the rationale behind *not* including `FileSaver` as a dependency. On top of that, saving files in Safari is in fact even more complicated and we ourselves resort to Flash in our application. So if we wanted to address saving files reliably, we would need to include a lot of functionality -- probably worth separating into another library (which breaks S in SOLID).

However, I agree that the documentation is misleading here, so we will fix it.
